cli_reference/openshift_developer_cli/odo-release-notes.adoc

```diff
@@ -82,14 +85,25 @@ Provide as many details in the issue description as possible.
 
 * link:https://bugzilla.redhat.com/show_bug.cgi?id=1761440[Bug 1761440] It is not possible to create two Services of the same type in one project.
 
+* link:https://bugzilla.redhat.com/show_bug.cgi?id=1821643[Bug 1821643] `odo push` does not work on the .NET component tag 2.1+.
++
+Workaround: specify your .NET project file by running:
++
+----
+$ odo config set --env DOTNET_STARTUP_PROJECT=<path to your project file>
+----
+
 [id="odo-technology-preview_{context}"]
-== Technology Preview features in `{odo-title}`{nbsp}{odo-ver}
+== Technology Preview features `{odo-title}` in `{odo-title}`{nbsp}{odo-ver}
 
-`odo debug` is a feature that allows users to attach a local debugger to a component running in the Pod on {product-title}. 
+* `odo debug` is a feature that allows users to attach a local debugger to a component running in the Pod. To learn more, see xref:../../cli_reference/openshift_developer_cli/debugging-applications-in-odo.adoc#debugging-an-application_debugging-applications-in-odo[Debugging applications in odo]
 
 :FeatureName: odo debug
 include::modules/technology-preview.adoc[leveloffset=+1]
 
-- Use the `odo debug port-forward` command to start port forwarding.
-- Use the `odo config set DebugPort 9292` command to specify the remote port that the debugging agent should run on.
-- Use the `odo debug port-forward --local-port 9292` command to specify the local port for port forwarding.
\ No newline at end of file
+* Devfile support. You can now create and deploy your applications using the devfiles with `{odo-title}`. A devfile is a file that defines the development environment: environment variables, image, and so on. To access this feature, you must enable experimental mode with `odo preference set experimental true`.
++
+To see the list currently supported devfile components, run `odo catalog list components`
```

Comment:
Missing the word "of". "To see the list **of** currently supported devfile components"...